docs/source/integrations/middleware.mdx

```diff
@@ -380,7 +380,21 @@ async function startApolloServer(typeDefs, resolvers) {
   });
 
   await server.start();
-  await server.applyMiddleware({ app });
+  await server.applyMiddleware({
+    app,
+    route: {            // shared route options for GET & POST requests
+      cors: true
```

Comment:
For consistency, this line and the "Get Route Description" line should end in commas